tests/test_models.py

```diff
@@ -58,8 +58,7 @@
 from tests.test_features import requires_piwik
 from tests.utils import mock_archive
 
-
-from tests.base import DEFAULT_METASCHEMA
+DEFAULT_METASCHEMA = get_default_metaschema()
```

Comment:
This method is called at import time, possibly before the test database is set up. As discussed in person, that may not be the intended behavior.